distributed/src/test/scala/asobu/distributed/service/RouteParamsExtractorSpec.scala

```diff
@@ -0,0 +1,40 @@
+package asobu.distributed.service
+
+import asobu.distributed.service.Extractors.RouteParamsExtractor
+import org.specs2.concurrent.ExecutionEnv
+import org.specs2.mutable.Specification
+import play.core.routing.RouteParams
+import shapeless._
+import shapeless.record._
+import asobu.dsl.CatsInstances._
+import concurrent.ExecutionContext.Implicits.global
+
+class RouteParamsExtractorSpec extends Specification {
+
+  "generates from Record T" >> {
+    type Rec = Record.`'x -> Int, 'y -> String, 'z -> Boolean`.T
+
+    val rpe = RouteParamsExtractor[Rec]
+    val result = rpe.run(RouteParams(Map("x" → Right("3")), Map.empty))
+    result.isLeft must beTrue
+
+    val result2 = rpe.run(RouteParams(Map("x" → Right("3"), "y" → Right("a"), "z" → Right("true")), Map.empty))
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/iHeartRadio/asobu/file/2261096903/issues/source?bid=3037259&fileBranchId=3178877#l21)